client/app/visualizations/components/EditVisualizationDialog.less

```diff
@@ -0,0 +1,18 @@
+@media (min-width: 992px) {
+  .edit-visualization-dialog {
+    display: flex;
+    height: 100%;
+
+    .visualization-settings {
+      padding: 0px 12px;
+      width: 40%;
+      overflow: auto;
+    }
+
+    .visualization-preview {
+      padding: 0px 12px;
```

Comment:
I noticed this after build completed, so: there should be no left padding for editor area and right padding for preview area (check the screenshot - editor is not aligned with dialog title):

![image](https://user-images.githubusercontent.com/12139186/75376813-5b3b9380-58d9-11ea-8103-755d4c75b659.png)